eventstore/test_tasks.py

```diff
@@ -0,0 +1,102 @@
+from django.test import TestCase
+
+from eventstore.models import CDUAddressUpdate
+from eventstore.tasks import read_data, upload_and_delete_cdu_address_updates
+
+
+class ProcessCDUExportTasks(TestCase):
+    def test_upload_and_delete_cdu_address_updates(self):
+        CDUAddressUpdate.objects.create(
+            msisdn="+2349030000000",
+            first_name="Jerry",
+            last_name="Cooney",
+            id_type="dob",
+            id_number="12345",
+            date_of_birth="1990-04-04",
+            folder_number="1234567",
+            district="Lagos",
+            municipality="Lagos",
+            city="Lagos",
+            suburb="Lekki",
+            street_name="Igbo Efun",
+            street_number="10045",
+        )
+
+        CDUAddressUpdate.objects.create(
+            msisdn="+2345555555555",
+            first_name="John",
+            last_name="Terry",
+            id_type="dob",
+            id_number="12123456",
+            date_of_birth="1982-05-04",
+            folder_number="2534567",
+            district="Lagos",
+            municipality="Lagos",
+            city="Lagos",
+            suburb="Lekki",
+            street_name="Igbo Efun",
+            street_number="10067",
+        )
+
+        upload_and_delete_cdu_address_updates(self)
```

Comment:
We need to have some assertions here to ensure that the file was created and uploaded properly.